csaf_2.1/json_schema/aggregator_json_schema.json

```diff
@@ -225,6 +198,18 @@
       "description": "Holds the date and time when the document was last updated.",
       "type": "string",
       "format": "date-time"
+    },
+    "maintained_from": {
```

Comment:
I don't really like the wording on this :(

Reply:
Me neither :-(